docker/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "2"
+
+services:
+  sparagn-media-service:
```

Comment:
As from previus comment I would prefer run the actual service in here with something lke:
```
command: ["./sparagn-media-service"]
```

We did the same for `sparagn-rest-service` and I think it is more consistennt with the port exposed from the docker-compose.yml file rather then from a Dockerfile.

Reply:
ports have to be exposed both. in the other project you're referring to the port was exposed and remapped in the docker-compose to the 80 which is not the case here coz if we had to run it in single machine (at early stage) we need another port available

Reply:
Ok, let's run the service from Dockerfile for now and expose the port from Dockerfile and from docker-compose